### PR TITLE
link-pointer class added to links inside text

### PR DESCRIPTION
**Pull request by peratik**
https://www.notion.so/santiment/Link-buttons-updates-ddb3cd3c912341a880e46a17b302bd35

### CODE REVIEW

File: src/ui/CookiesPopup.svelte [hunk 1]
```diff
@@ -37,8 +37,9 @@
     <h2 class="body-2 txt-m mrg-s mrg--b">We are using cookies to improve your experience!</h2>
     <h4 class="body-3 mrg-xl mrg--b c-waterloo">
       By clicking “Allow all”, you agree to use of all cookies. Visit our
-      <a href="https://santiment.net/cookies/" target="_blank" class="c-accent">Cookies Policy</a> to
-      learn more.
+      <a href="https://santiment.net/cookies/" target="_blank" class="c-accent link-pointer"
```
Comment (DmitriVanGuard) [inline src/ui/CookiesPopup.svelte:40]: `c-accent` is redundant

File: src/ui/LoginPrompt/Section.svelte [hunk 2]
```diff
@@ -23,7 +23,7 @@
   {#if bottomLabel}
     <div class="body-2 c-waterloo mrg-xl mrg--t">
       {bottomLabel}
-      <a href={bottomHref} class="btn" sapper:prefetch>{bottomAction}</a>
+      <a href={bottomHref} class="btn link-pointer" sapper:prefetch>{bottomAction}</a>
```
Comment (DmitriVanGuard) [inline src/ui/LoginPrompt/Section.svelte:26]: `btn` is redundant

File: src/ui/LoginPrompt/Metamask.svelte [hunk 3]
```diff
@@ -24,14 +24,15 @@
...
+    style="--url:url({metamaskInverseSvg})"
+  >
     Log in with Metamask
   </div>
 {:else}
   <div class="metamask" style="--url:url({metamaskSvg})">
     <h3 class="body-2 txt-m mrg-xs mrg--b">We can't detect Metamask!</h3>
     <p class="mrg-s mrg--b">We can auth you with Metamask account. It's secure and easy.</p>
-    <a href="https://metamask.io/#how-it-works" class="btn">How Metamask works?</a>
+    <a href="https://metamask.io/#how-it-works" class="btn link-pointer">How Metamask works?</a>
```
Comment (DmitriVanGuard) [inline src/ui/LoginPrompt/Metamask.svelte:35]: `btn` is redundant

File: src/ui/ManageCookiesDialog/index.svelte [hunk 4]
```diff
@@ -56,30 +56,37 @@
     visited. The information does not usually directly identify you, but it can give you a safe and
     more personalized web experience. Because we respect your right to privacy, you can choose not
     to allow some types of cookies. Blocking some types of cookies may impact your experience on the
-    site. <a href="https://santiment.net/cookies/" target="_blank" class="c-accent">Learn more</a>
+    site. <a href="https://santiment.net/cookies/" target="_blank" class="c-accent link-pointer"
```
Comment (DmitriVanGuard) [inline src/ui/ManageCookiesDialog/index.svelte:59]: `c-accent` is redundant

File: src/ui/Pricing/SubscriptionSettings/SubscriptionCard/PlanCard.svelte [hunk 5]
```diff
@@ -70,8 +70,10 @@
       {/if}
 
       Check all plans
-      <a href="https://app.santiment.net/pricing" class="btn-0" on:click={window.__onLinkClick}
-        >here!</a
+      <a
+        href="https://app.santiment.net/pricing"
+        class="btn-0 link-pointer"
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/SubscriptionSettings/SubscriptionCard/PlanCard.svelte:75]: `btn-0` is redundant

File: src/ui/Pricing/SubscriptionSettings/SubscriptionCard/UserPlanCard.svelte [hunk 6]
```diff
@@ -46,8 +46,10 @@
       {/if}
     {:else}
       Starter plan with limited access to Sanbase features. Check all plans
-      <a href="https://app.santiment.net/pricing" class="btn-0" on:click={window.__onLinkClick}
-        >here!</a
+      <a
+        href="https://app.santiment.net/pricing"
+        class="btn-0 link-pointer"
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/SubscriptionSettings/SubscriptionCard/UserPlanCard.svelte:51]: `btn-0` is redundant

File: src/ui/Products/Products.svelte [hunk 7]
```diff
@@ -14,7 +14,7 @@
   {#if isColumn}
     <div class="more body-2 c-waterloo mrg-xl mrg--t">
       To learn more about Santiment's products, team, or available jobs, visit <a
-        class="btn-0"
+        class="btn-0 link-pointer"
```
Comment (DmitriVanGuard) [inline src/ui/Products/Products.svelte:17]: `btn-0` is redundant

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

File: src/ui/LoginPrompt/Section.svelte [hunk 8]
```diff
@@ -23,7 +23,7 @@
   {#if bottomLabel}
     <div class="body-2 c-waterloo mrg-xl mrg--t">
       {bottomLabel}
-      <a href={bottomHref} class="btn" sapper:prefetch>{bottomAction}</a>
+      <a href={bottomHref} class="link-pointer" sapper:prefetch>{bottomAction}</a>
```
[the remark this answers is not in the document]
Comment (peratik) [reply src/ui/LoginPrompt/Section.svelte:26]: There is another `a` tag:

[the remark this answers is not in the document]
Comment (peratik) [reply src/ui/LoginPrompt/Section.svelte:26]:
```
  {#if $IsMobile}
    <a href="/" class="close btn"><Svg id="close" w="16" /></a>
  {/if}
```

File: src/ui/PaymentDialog/Banner.svelte [hunk 9]
```diff
@@ -35,7 +35,9 @@
       Enjoy your 14-day free trial of Sanbase {name || ''}!
     {:else if trialDaysLeft < 1}
       Your Pro trial has expired! If you have accidentally bypassed the free trial, please get in
-      touch with <button class="btn c-accent" on:click={showIntercom}>our support team</button>
+      touch with <button class="btn c-accent link-pointer" on:click={showIntercom}
```
[the remark this answers is not in the document]
Comment (peratik) [reply src/ui/PaymentDialog/Banner.svelte:38]: 1. button links also included

[the remark this answers is not in the document]
Comment (peratik) [reply src/ui/PaymentDialog/Banner.svelte]: 2. Fixed

File: src/ui/LoginPrompt/Section.svelte (hunk 8 above)
[the remark this answers is not in the document]
Comment (peratik) [reply src/ui/LoginPrompt/Section.svelte:26]: Fixed

**Review by DmitriVanGuard — APPROVED** [review]

**Pull request merged**